examples/byte-io/byte_io.description

```diff
@@ -1,3 +1,3 @@
-//This example demonstrates how bytes can be read and written through the I/O API.<br/><br/>
+//This example demonstrates how bytes can be read and written through the I/O APIs.<br/><br/>
```

Comment:
Add a space after "//" - even though this wouldn't change the final rendered output though, nice to keep it tidy.

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1090/commits/896e19761b52ecd694d670843f0eae5f6d69695a